packages/fuels-programs/src/contract.rs

```diff
@@ -693,6 +719,29 @@ impl<T: Account> MultiContractCallHandler<T> {
             .map_err(|err| map_revert_error(err, &self.log_decoder))
     }
 
+    pub async fn call_call_with_retry<D: Tokenizable + Debug>(
```

Comment:
why not just `call_with_retry`? For `MultiContractCallHandler` we use the `call` method, the same as `ContractCallHandler` so I don't see why we would have different names here.

Reply:
Probably a paste-ing mishap.